tests/ert3/engine/integration/conftest.py

```diff
@@ -3,6 +3,8 @@
 import pytest
 import numpy as np
 
+import ert3
```

Comment:
Isn't it possible to only import `ert3.workspace` here and noot the entire module? 
```suggestion
import ert3.workspace
```

Reply:
Good idea 👏